src/power_divergence.jl

```diff
@@ -214,6 +235,48 @@ end
 
 # Under regularity conditions, their asymptotic distributions are all the same (Drost 1989)
 # Chi-squared null approximation works best for lambda near 2/3
+"""
+    PowerDivergenceTest(x [,y] [, lambda] [,theta0] )
```

Comment:
Spaces must go after commas. Also mention the default values of `lambda` and `theta`. It could make sense to use two lines for the signature, one without `y` and one with it.